lua/wikis/chess/ChessEco.lua

```diff
@@ -0,0 +1,32 @@
+---
+-- @Liquipedia
+-- wiki=chess
+-- page=Module:ChessEco
```

Comment:
Can we rename "eco" to something else? Like "Openings"? 

Eco is often use as an abbr for "Economy".

Reply:
I assume Eco was refer to this one but in coding  context I can probably change that to ChessOpenings or something 
![image](https://github.com/user-attachments/assets/70a14062-5461-42b1-94a7-ca7e3bd883ab)